src/contradictory_claims/cli.py

```diff
@@ -92,7 +94,48 @@ def main(train, report):
     covid19_filt_section_df = covid19_df.loc[covid19_df.section.str.lower().isin(section_list)]
 
     # Clean the text to keep only meaningful sentences
-    covid19_clean_df = clean_text(covid19_filt_section_df)  # noqa: F841
+    # and merge sentences belonging to each section of each paper into contiguous text passages
+    covid19_clean_df = clean_text(covid19_filt_section_df)
+
+    # Merge all sentences belonging to each section of each paper into contiguous text passages
+    covid19_merged_df = merge_section_text(covid19_clean_df)
+
+    # Construct regex match pattern for drug terms
+    # Flank drug terms with white space for accurate match
+    with open(drug_lex_path) as f:
+        drug_terms = f.read().splitlines()
+    drug_terms.append('acei/arb')  # TODO: Add this to the main lexicon file
```

Comment:
Yeah I'd thought about this initially too. I'd thought it would better to keep some of the simpler code in there instead of pushing everything to functions and thus defeating the purpose of functions - one place for code that's used in multiple places. But I think I agree with you now, cli.py is getting a bit longer and pushing to functions would make things bit neater.